.gitignore

```diff
@@ -111,6 +111,7 @@ dockers/docker-fpm-frr/Dockerfile
 dockers/docker-fpm-gobgp/Dockerfile
 dockers/docker-fpm-quagga/Dockerfile
 dockers/docker-lldp-sv2/Dockerfile
+dockers/docker-namespace/Dockerfile
```

Comment:
why keep it?

Reply:
this is a .gitignore file

Reply:
but we no longer build docker-namespace docker, right?

Reply:
deleted docker-namespace related file from gitignore